src/check_cron_runtime.py

```diff
@@ -24,11 +24,28 @@
 from subprocess import Popen, PIPE, STDOUT
 from argparse import ArgumentParser
 from sys import exit
+from os.path import isfile
 
-# A list of cron names that shall get ignored by this check.
-exclude_list = [
-    'am_stockpile_distribution',
-]
+
+# Path to the file which has a list of cron names (line separated)
+# that shall get ignored by this check
+exclude_file = '/etc/check_cron_runtime_exclude'
+
+
+def parse_exclude_file():
+    """Parses the exclude file and and adds given list
+
+    Returns a combined list
+    """
+
+    excluded_crons = []
+
+    if isfile(exclude_file):
+        with open(exclude_file) as fd:
+            for cron in list(filter(None, [line.rstrip('\n') for line in fd])):
+                excluded_crons.append(cron)
+
+    return excluded_crons
```

Comment:
Actually we can make it shorter, as we now already get the list we want:
```suggestion
    if not isfile(exclude_file):
        return []
    
    with open(exclude_file) as fd:
            return list(filter(None, [line.rstrip('\n') for line in fd]))
```